### DISCUSSION TITLE
Question on WEBRTC Setting

### DISCUSSION THREAD

**Discussion by tbrausch** [mrlt8/docker-wyze-bridge#414] [Q&A] [answered]
Just a quick question.   I currently use the WebRTC Card integration to display cameras in home assistant.  I have searched both issues and discussions to find out about the WEBRTC setting in Docker Wyze Bridge but have not been able to find any info on what this does or how to set it up.  If I select thissetting, does it mean I can use the embedded webrtc support in HA rather than the WebRTC ard?  If so, are there benefits to this and how do I set it up?

**Comment by mrlt8** [comment, marked as answer]
At this time, the `WEBRTC` setting will only pull the Amazon kinesis viewing credentials for each camera. Basically the same if not identical webrtc stream that the wyze web view uses.

It's been a while since I've looked into webrtc support in HA, but I believe most of the webrtc stuff was to convert and view rtsp streams as webrtc and not to view native webrtc streams?
